op-node/sources/types.go

```diff
@@ -115,6 +118,9 @@ type rpcHeader struct {
 	// BaseFee was added by EIP-1559 and is ignored in legacy headers.
 	BaseFee *hexutil.Big `json:"baseFeePerGas" rlp:"optional"`
 
+	// WithdrawalsRoot was added by EIP-4895 and is ignored in legacy headers.
+	WithdrawalsRoot *common.Hash `json:"withdrawalsRoot" rlp:"optional"`
```

Comment:
Do we every RLP encode or decode this struct? I know we already have the rlp tags, but I don't think we need them.